bot/utils/blurple_formatter.py

```diff
@@ -0,0 +1,711 @@
+            node.name,
+            node.args,
+            node.body,
+            node.decorator_list,
+            node.returns,
+        )
+
+        return "\n".join(
+            [
+                *(f"@{space()}{unparse(decorator)}" for decorator in decorator_list),
+                f"{f'sync{space(1)}'*(random.random()>.5)}def{space(1)}{name}{space()}"
+                f"({space()}{unparse(args, True)}{space()})"
+                + (f"{space()}->{space()}{unparse(returns)}" if returns else "")
+                + f"{space()}:",
+                *indent(body, randrange(2, 5)),
+            ]
+        )
+    if isinstance(node, ast.GeneratorExp):
+        elt, generators = node.elt, node.generators
+
+        return (
+            f"({space()}"
+            + f"{unparse(elt)}{space()}"
+            + f"{space()}".join(map(unparse, generators))
+            + f"{space()})"
+        )
+    if isinstance(node, ast.Global):
+        names = node.names
+
+        return f"global{space(1)}{', '.join(map(unparse, names))}{space()};"
+    if isinstance(node, ast.If):
+        test, body, orelse = node.test, node.body, node.orelse
+
+        s = [
+            f"if{space(1)}{unparse(test)}{space()}:",
+            *indent(body, randrange(2, 5)),
+        ]
+        if orelse:
+            s.append(f"else{space()}:")
+            s.extend(indent(orelse, randrange(2, 5)))
+        return "\n".join(s)
+    if isinstance(node, ast.IfExp):
+        test, body, orelse = node.test, node.body, node.orelse
+
+        return (
+            f"{(parenthesize if precedences[type(body)] < precedences[ast.IfExp] else unparse)(body)}{space(1)}"
+            + f"if{space(1)}"
+            + f"{(parenthesize if precedences[type(body)] < precedences[ast.IfExp] else unparse)(test)}{space(1)}"
+            + f"else{space(1)}{(parenthesize if precedences[type(body)] < precedences[ast.IfExp] else unparse)(orelse)}"
+        )
+    if isinstance(node, ast.Import):
+        names = node.names
+
+        return (
+            f"import{space(1)}"
+            + f",{space()}".join(
+                f"{name.name}"
+                + (f"{space(1)}as{space(1)}{name.asname}" if name.asname else "")
+                for name in names
+            )
+            + f"{space()};"
+        )
+    if isinstance(node, ast.ImportFrom):
+        module, names, level = node.module, node.names, node.level
+
+        return (
+            f"from{space(1)}{'.'*level}{module}{space(1)}import{space(1)}"
+            + f",{space()}".join(
+                f"{name.name}"
+                + (f"{space(1)}as{space(1)}{name.asname}" if name.asname else "")
+                for name in names
+            )
+            + f"{space()};"
+        )
+    if isinstance(node, ast.JoinedStr):
+
+        return ast.unparse(node)
+    if isinstance(node, ast.Lambda):
+        args, body = node.args, node.body
+
+        return f"lambda{space(1)}{unparse(args)}{space(1)}:{space()}{unparse(body)}"
```

Comment:
```suggestion
        return f"lambda{space(1)}{unparse(args)}{space()}:{space()}{unparse(body)}"
```
I don't think at least one space is required there? `lambda arg1, arg2: arg2` for example.